addon/ng2/commands/get.ts

```diff
@@ -0,0 +1,29 @@
+import * as chalk from 'chalk';
+import * as Command from 'ember-cli/lib/models/command';
+import {CliConfig} from '../models/config';
+
+
+const GetCommand = Command.extend({
+  name: 'get',
+  description: 'Set a value in the configuration.',
+  works: 'insideProject',
```

Comment:
If it is possible to have a global config, this should command should work even outside of projects.